docker/recorder/src/test/java/org/arquillian/cube/docker/impl/client/reporter/TakeDockerEnvironmentTest.java

```diff
@@ -131,4 +165,48 @@ public void should_report_schema_of_docker_composition() {
         assertThat(screenshotEntry.getLink()).isEqualTo("reports/schemas/docker_composition.png");
     }
 
+   @Test
+    public void should_report_log_file(){
+        final TakeDockerEnvironment takeDockerEnvironment = new TakeDockerEnvironment();
+        takeDockerEnvironment.propertyReportEvent = propertyReportEvent;
+        takeDockerEnvironment.reportContainerLogs(new BeforeStop(CUBE_ID), dockerClientExecutor, new ReporterConfiguration());
+        verify(propertyReportEvent).fire(propertyReportEventArgumentCaptor.capture());
+
+        final PropertyReportEvent propertyReportEvent = propertyReportEventArgumentCaptor.getValue();
+        final PropertyEntry propertyEntry = propertyReportEvent.getPropertyEntry();
+        assertThat(propertyEntry).isInstanceOf(FileEntry.class);
+
+        FileEntry fileEntry = (FileEntry) propertyEntry;
+        assertThat(fileEntry.getPath()).isEqualTo("target/reports/logs/tomcat.log");
+    }
+
+    @Test
```

Comment:
I would say any logic we introduce should be covered by tests. Ideally test-first. Otherwise how do you know it works? From logs? From user error reports?